_posts/2024-01-19-yannick-horrix-opentofu.md

```diff
@@ -0,0 +1,160 @@
+Terraform configuration, on the other hand, can be more verbose and complex to maintain.
+## Licensing and Pricing
+Both Terraform and OpenTofu are open-source tools, providing free usage.
+
+As said, OpenTofu uses the Mozilla Public License (MPL), which allows developers to freely use, modify, and distribute software for both commercial and non-commercial use.
+
+Terraform has now shifted to the Business Source License (BUSL). This also permits free use of source code, including commercial use, except when it’s used to provide an offering that competes with HashiCorp.
+An example of this could be running Terraform in a hosted way in CI/CD and offering this as a production service.
+
+So while, Terraform remains free to use in your projects, there can be an issue when incorporating Terraform in a service offering.
+
+In terms of paid offerings, Terraform does offer multiple options with Terraform Cloud, a platform managing Terraform code, availaible as a hosted service. 
+Additionally, organizations with advanced security and compliance needs can purchase Terraform Enterprise. 
+This is a self-hosted distribution of Terraform Cloud which offers a private instance that includes the advanced features available in Terraform Cloud.
+
+## Community and Support
+Terraform currently holds the advantage when it comes to community support. 
+The Terraform community is vast, offering a large breadth of resources. 
+This means that users have more readily accessible examples at their disposal and can find solutions to their problems more quickly.
+
+OpenTofu, on the other hand, is a growing tool with an expanding community. 
+Although not as large as Terraform's, its community is rapidly expanding as more users move to OpenTofu. 
+This will provide ever increasing support and resources for the tool in the future.
+## Maturity and Stability
+Terraform is proven as a mature, stable and extremely popular orchestration tool capable of handling enterprise-grade infrastructure deployments.
+OpenTofu, as a fork of Terraform, is a newer tool with a relatively newer and may be more susceptible to bugs and instability. However, the “OpenTofu” community is actively addressing these issues, and its stability is expected to improve with time.
+## Summary
+To summarize, the following table captures the comparison between the two tools.
+
+| Feature | OpenTofu | Terraform |
+|----------|----------|----------|
+| Features | Similar to Terraform, with some improvements | Large featureset |
+| Configuration | Declarative, easier to use | Procedural, more complex |
+| Licensing | Mozilla Public License (MPL), open-source | Business Software License (BUSL), open-source with restrictions |
+| Pricing | Free to use | Free to use, with some restrictions for commercial use |
+| Community | Smaller, but expanding | Large |
+| Maturity | New | Mature |
```

Comment:
Add 
| Ownership | Part of Linux Foundation | Owned by Hashicorp |
| Open Source | Yes | Source-available |